### PR TITLE
adding LD matrices and scores script

### PR DESCRIPTION
**Pull request by konradjk**

### CODE REVIEW

File: annotations/generate_ld_data.py [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+                           (ht.pop_freq.AN / n >= 2 * call_rate_cutoff)).add_index(name='new_idx')
+
+            indices = ht.idx.collect()
+
+            r2 = BlockMatrix.read(ld_matrix_path(data_type, pop))
+            r2 = r2.filter(indices, indices) ** 2
+            r2_adj = ((n - 1.0) / (n - 2.0)) * r2 - (1.0 / (n - 2.0))
+
+            starts_and_stops = hl.linalg.utils.locus_windows(ht.locus, radius, _localize=False)
+            r2_adj = BlockMatrix._from_java(r2_adj._jbm.filterRowIntervalsIR(
```
Comment (lfrancioli) [inline annotations/generate_ld_data.py]: Needs a comment

File: annotations/generate_ld_data.py [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+    pop_mt = pop_mt.annotate_rows(pop_freq=pop_mt.freq[meta_index])
+    pop_mt = pop_mt.filter_rows((hl.len(pop_mt.filters) == 0) &
+                                (pop_mt.pop_freq.AC > 1) &
+                                (pop_mt.pop_freq.AN - pop_mt.pop_freq.AC > 1) &
+                                ~((pop_mt.pop_freq.AF == 0.5) &
+                                  (pop_mt.pop_freq.homozygote_count == 0)))
+    return pop_mt
+
+
+def import_reference_ld_scores():
```
Comment (lfrancioli) [inline annotations/generate_ld_data.py]: This is never called -- is it meant to be imported then used? If so is that the best place for it?

File: annotations/generate_ld_data.py [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+
+def ld_pruned_path(data_type: str, pop: str, r2: str, version: str = CURRENT_RELEASE):
+    return f'gs://gnomad-resources/ld/gnomad.{data_type}.r{version}.{pop}.ld.pruned_set.r2_{r2}.ht'
+
+
+def ld_scores_path(data_type: str, pop: str, version: str = CURRENT_RELEASE):
+    return f'gs://gnomad-resources/ld/gnomad.{data_type}.r{version}.{pop}.ld_scores.ht'
+
+
+eur_w_ld_chr_ht = 'gs://konradk/ld_score/reference_data/eur_w_ld_chr/20.l2.ldscore.ht'
```
[the remark this answers is not in the document]
Comment (konradjk) [reply annotations/generate_ld_data.py]: Oh yeah, I can get rid of these - these were for my own purposes to correlate the old LD scores with new ones.

File: annotations/generate_ld_data.py [hunk 4]
```diff
@@ -0,0 +1,165 @@
+from gnomad_hail import *
+from hail.utils import new_temp_file
+from hail.utils.java import Env
+from hail.linalg import BlockMatrix
+
+
+def ld_matrix_path(data_type: str, pop: str, version: str = CURRENT_RELEASE):
```
[the remark this answers is not in the document]
Comment (konradjk) [reply annotations/generate_ld_data.py]: They'll eventually go in `gnomad-public` but wanted to have a short discussion on where exactly before making that move

**Pull request merged**